recipes/tensorstore/meta.yaml

```diff
@@ -0,0 +1,112 @@
+{% set name = "tensorstore" %}
+{% set version = "0.1.21" %}
+# Bazel version is specified upstream, file .bazelversion
+{% set bazel_version = "5.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/tensorstore-{{ version }}.tar.gz
+  sha256: c620c3b06e1606092bd444bb7687c9d5a8049e8ed5f4c17ee0fd53e0a43b7df7
+  patches:
+    # Looks like they forgot to add update outputs in nasm and xz
+    - patches/system_nasm.patch
+    - patches/system_xz.patch
+    # Can't use keyword arguments in ctx.template()
+    - patches/repo_template.patch
+    # we are using openssl instead of boringssl (Google's fork)
+    # it mostly works out of the box, but we need to make sure the right symbol is used
+    # we are commenting out an include that doesn't exist on openssl
+    # and also replacing an old function name with its modern equivalent-ish (_cleanup->_reset)
+    - patches/oauth-boringssl.patch
+    # These two patches are FUN: we patch the workspace script from bazel
+    # so it patches the half.hpp header it will download with our meta-patch :D
+    # Idea for patch coming from # https://sourceforge.net/p/half/discussion/general/thread/86298c105c/
+    - patches/net_sourceforge_half.patch
+    - patches/half.hpp.patch.patch
+    # Use conda-forge's nlohmann_json
+    - patches/nlohmann_json-workspace.patch
+    - patches/nlohmann_json-build.patch
+    # Use conda-forge's pybind11
+    - patches/pybind11-system-build.patch
+    - patches/pybind11-workspace.patch
+    # MacOS needs an extra linker flag so pybind11 finds the python symbols in _tensorstore.so :shrug:
+    - patches/pybind11-linkopts.patch  # [osx]
+
+build:
+  skip: true  # [py<37 or win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    # Keep bazel listed twice here to help the migrators track dependencies
+    - bazel
+    - bazel {{ bazel_version }}
+    - bazel-toolchain
+  host:
+    - pip
+    - python
+    - wheel
+    - setuptools-scm
+    - numpy
+    - libprotobuf
```

Comment:
Link inspector says that libprotobuf isn't used.

Reply:
Without it, the build system [fails to start](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=534276&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2105) (line 2105):

```
  /home/conda/.cache/bazel/_bazel_conda/install/68c8a1420cd851bd87446b4ab379485e/process-wrapper: error while loading shared libraries: libprotobuf.so.30: cannot open shared object file: No such file or directory
```

Maybe I should only add it to the `build` section though.